### PR TITLE
Added 4 RDS and 2 KMS checks.

### PR DESCRIPTION
**Pull request by jodzga**

### CODE REVIEW

File: aws_quota/check/rds.py [hunk 1]
```diff
@@ -49,3 +67,66 @@ class RDSEventSubscriptions(QuotaCheck):
...
+    quota_code = "L-732153D0"
+    used_services = [service_code]
+
+    @property
+    def current(self) -> int:
+        return len(get_all_db_security_groups(self.get_client(self.service_code)))
+
+
+class RDSCheck(QuotaCheck):
+    key = "rds_security_groups"
```
Comment (yanbinren) [inline aws_quota/check/rds.py]: Should we use a different class name and key for this? e.g. rds_security_groups_vpc

File: aws_quota/check/rds.py [hunk 2]
```diff
@@ -49,3 +67,66 @@ class RDSEventSubscriptions(QuotaCheck):
...
+    quota_code = "L-36B04611"
+    used_services = [service_code]
+
+    @property
+    def current(self) -> int:
+        return len(list(filter(lambda sg: 'VpcId' in sg and sg['VpcId'] is not None, get_all_db_security_groups(self.get_client(self.service_code)))))
+
+
+class RDSCheck(InstanceQuotaCheck):
+    key = "rds_security_groups"
```
Comment (yanbinren) [inline aws_quota/check/rds.py]: What is this check? It seems to be a duplicate of RDSReadReplicasPerMasterCheck?

File: aws_quota/check/rds.py (hunk 1 above)
Comment (jodzga) [reply aws_quota/check/rds.py]: Ups, pushed some intermediate state of the branch by accident...
Good catch!

File: aws_quota/check/rds.py (hunk 2 above)
Comment (jodzga) [reply aws_quota/check/rds.py]: Ups, pushed some intermediate state of the branch by accident...
Good catch!

File: aws_quota/check/rds.py [hunk 3]
```diff
@@ -49,3 +67,66 @@ class RDSEventSubscriptions(QuotaCheck):
...
+    quota_code = "L-732153D0"
+    used_services = [service_code]
+
+    @property
+    def current(self) -> int:
+        return len(get_all_db_security_groups(self.get_client(self.service_code)))
+
+
+class RDSSecurityGroupsVPCCheck(QuotaCheck):
+    key = "rds_security_groups_vpv"
```
Comment (yanbinren) [inline aws_quota/check/rds.py]: vpv -> vpc

**Review by yanbinren — APPROVED** [review]

**Pull request merged**